meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -498,6 +498,25 @@ SOTA_EXTRA_CLIENT_FEATURES:append:sota:uz3eg-iocc-sec = " fiovb"
 UBOOT_SPL_SIGN_ENABLE:sota:uz3eg-iocc-ebbr-sec = "1"
 UBOOT_SIGN_ENABLE:sota:uz3eg-iocc-ebbr-sec = "0"
 
+## Xilinx Versal
+PREFERRED_PROVIDER_u-boot:versal = "u-boot-xlnx"
+PREFERRED_PROVIDER_u-boot-fw-utils:versal = "libubootenv"
+PREFERRED_RPROVIDER_u-boot-fw-utils:versal = "libubootenv"
+SOTA_CLIENT_FEATURES:append:versal = " ubootenv"
+WKS_FILE_DEPENDS:append:sota:versal = " u-boot-default-script"
+PREFERRED_PROVIDER_u-boot-default-script:versal = "u-boot-base-scr"
+PREFERRED_PROVIDER_u-boot-default-script:sota:versal = "u-boot-ostree-scr-fit"
+PREFERRED_PROVIDER_virtual/kernel:versal = "linux-lmp-xlnx"
+PREFERRED_PROVIDER_virtual/dtb:versal = "device-tree"
+PREFERRED_PROVIDER_virtual/trusted-firmware-a:versal = "arm-trusted-firmware"
+KERNEL_BUILTIN_WIREGUARD:versal = "1"
+EXTRA_IMAGEDEPENDS:remove:versal = "qemu-devicetrees"
+UBOOT_ENTRYPOINT:versal = "0x18000000"
+OSTREE_KERNEL_ARGS:versal ?= "console=ttyAMA0,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
+KERNEL_IMAGETYPE:sota:versal = "fitImage"
+KERNEL_CLASSES:sota:versal = " kernel-lmp-fitimage "
```

Comment:
is this line missing an `:append` or the extra space are only extra? (just double checking)

Reply:
Don't think we need the extra space, just a copy & paste issue.

Reply:
Removed.